src/commands/help.js

```diff
@@ -8,8 +8,8 @@ import {
   BROADCAST_ENABLED,
 } from '../helpers/constants'
 const webAppEnabled =
-  WEB_APPLICATION.value === 'yes' && WEB_INTERFACE.value === 'yes'
-const mapEnabled = BOT_MAPS.value === 'yes'
+  WEB_APPLICATION?.value === 'yes' && WEB_INTERFACE?.value === 'yes'
```

Comment:
this is probably my fav part of the newest additions to es6. But it does present some problems sometimes, evaluating true because of the variable before ?.